compiler/damlc/daml-prim-src/GHC/Show.daml

```diff
@@ -121,6 +121,11 @@ instance Show Decimal where
   show = primitive @"BEToText"
 #endif
 
+#ifdef DAML_UNSTABLE
+instance Show BigDec where
+  show = primitive @"BEToText"
+#endif
```

Comment:
Likewise, we could just implement `Show BigDecimal` where we use the primitive directly. Even more so since that `Show` instance anyway adds the rounding mode